### PR TITLE
feature(kensetsu): kgold as stablecoin

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Sets KGOLD as stablecoin for Kensetsu.

### CODE REVIEW

**Review by ra9mls — APPROVED** [review]

File: node/chain_spec/src/lib.rs [hunk 1]
```diff
@@ -1675,10 +1675,13 @@ fn testnet_genesis(
         },
         iroha_migration: iroha_migration_config,
         kensetsu: KensetsuConfig {
-            predefined_stablecoin_infos: vec![
+            predefined_stablecoin_sora_peg: vec![
                 (KUSD, DAI, balance!(1)),
                 (KXOR, XOR, balance!(100000))
             ],
+            predefined_stablecoin_oracle_peg: vec![
+                (KGOLD, SymbolName::xau(), 1000000000000000),
```
Comment (wer1st) [inline node/chain_spec/src/lib.rs:1683]: More readable
```suggestion
                (KGOLD, SymbolName::xau(), balance!(0.001)),
```

File: pallets/kensetsu/src/migrations.rs [hunk 2]
```diff
@@ -256,6 +256,19 @@ pub mod v1_to_v2 {
...
                 weight += <T as frame_system::Config>::DbWeight::get().writes(1);
 
+                StablecoinInfos::<T>::insert(
+                    AssetIdOf::<T>::from(KGOLD),
+                    StablecoinInfo {
+                        bad_debt: balance!(0),
+                        stablecoin_parameters: StablecoinParameters {
+                            peg_asset: PegAsset::OracleSymbol(SymbolName::xau()),
+                            // approximately ~$4
+                            minimal_stability_fee_accrue: 1000000000000000,
```
Comment (wer1st) [inline pallets/kensetsu/src/migrations.rs:266]:
```suggestion
                            minimal_stability_fee_accrue: balance!(0.001),
```

File: pallets/kensetsu/src/migrations.rs [hunk 3]
```diff
@@ -363,6 +376,16 @@ pub mod v1_to_v2 {
                     kusd_info.stablecoin_parameters
                 );
 
+                let kgold_info = StablecoinInfos::<TestRuntime>::get(KGOLD).unwrap();
+                assert_eq!(balance!(0), kgold_info.bad_debt);
+                assert_eq!(
+                    StablecoinParameters {
+                        peg_asset: PegAsset::OracleSymbol(SymbolName::xau()),
+                        minimal_stability_fee_accrue: 1000000000000000,
```
Comment (wer1st) [inline pallets/kensetsu/src/migrations.rs:384]:
```suggestion
                        minimal_stability_fee_accrue: balance!(0.001),
```

File: pallets/kensetsu/src/migrations.rs [hunk 4]
```diff
@@ -256,6 +256,19 @@ pub mod v1_to_v2 {
                 );
                 weight += <T as frame_system::Config>::DbWeight::get().writes(1);
 
+                StablecoinInfos::<T>::insert(
+                    AssetIdOf::<T>::from(KGOLD),
```
Comment (wer1st) [inline pallets/kensetsu/src/migrations.rs:260]: I don't remember, is KGOLD already registered as asset in some previous migration?

**Review by vovac12 — APPROVED** [review]

**Review by wer1st — APPROVED** [review]

**Pull request merged**